source/_patterns/00-base/_init.scss

```diff
@@ -42,7 +42,9 @@ nav,
 // General styles (TODO: most likely move those to DB UI Base)
 :root {
 	--db-focus-outline-offset: 1px;
-	--db-focus-outline-color: #2c95c4;
+	--db-focus-outline-color: #0087b9;
+
+	--db-disabled-text-opacity: 0.5;
```

Comment:
what about moving those to DB UI Base ?

Reply:
as discussed this change will be done separately with #89